scripts/bootstrap-cluster.sh

```diff
@@ -0,0 +1,99 @@
+#!/bin/bash
+
+usage() {
+    cat <<-EOF
+usage: bootstrap-cluster.sh
+      -d docker
+      -t trust_store_password
+      -k key_store_password      
+EOF
+}
+
+update_properties() {
+  local trust_pass="$1"
+  local key_pass="$2"
+  local properties_file="./mm2.properties"
+
+  echo >> "$properties_file"
+  echo "security.protocol = SSL" >> "$properties_file"
+  echo "ssl.endpoint.identification.algorithm =" >> "$properties_file"
+  echo "ssl.truststore.location = /bitnami/kafka/config/certs/truststore.jks" >> "$properties_file"
+  echo "ssl.truststore.password =" "$trust_pass" >> "$properties_file"
+  echo "ssl.truststore.type = JKS" >> "$properties_file"
+  echo "ssl.keystore.location = /bitnami/kafka/config/certs/mm.keystore.jks" >> "$properties_file"
+  echo "ssl.keystore.password =" "$key_pass" >> "$properties_file"
+
+  cat $properties_file
+}
+
+kill_kafka() {
```

Comment:
Maybe use `kafka_mirror` or `kafka_connect` for naming

Reply:
fixed